src/NCronJob/Configuration/JobOptionBuilder.cs

```diff
@@ -39,6 +40,50 @@ public ParameterBuilder WithCronExpression(string cronExpression, bool? enableSe
         return new ParameterBuilder(this, jobOption);
     }
 
+    /// <summary>
+    /// Configures the job to run once during the application startup before any other jobs.
+    /// </summary>
+    /// <returns>Returns a <see cref="ParameterBuilder"/> that allows adding parameters to the job.</returns>
+    public ParameterBuilder RunAtStartup()
+    {
+        var jobOption = new JobOption
+        {
+            IsStartupJob = true
+        };
+
+        jobOptions.Add(jobOption);
+
+        return new ParameterBuilder(this, jobOption);
+    }
+
+
+    /// <summary>
+    /// Configures the job to run once after a specified delay.
+    /// </summary>
+    /// <param name="delay">The delay after which the job should be executed.</param>
+    /// <returns>Returns a <see cref="ParameterBuilder"/> that allows adding parameters to the job.</returns>
+    public ParameterBuilder RunOnce(TimeSpan delay)
```

Comment:
See [#43 ](https://github.com/NCronJob-Dev/NCronJob/issues/43#issuecomment-2125618158)

I somewhat doubt that we really need the delay.
I am trying to come up with a use-case but really can't.

Reply:
No worries, I'll remove it. You still want the one that passes absoluteDateTime?

Reply:
I would argue that is even less likely than a delay :D
But that is just my current horizon. If a user wants that, we still can add that later on.

Reply:
RunOnce was removed